### PR TITLE
feat [C4R-832] Practice programs list

### PR DESCRIPTION
**Pull request by AndreeaMariaP**
[C4R-832](https://wearetribus.atlassian.net/browse/C4R-832)
Created the practice programs list

### CODE REVIEW

File: src/common/helpers/format.helper.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { format } from 'date-fns';
```
Comment (dragos1195) [inline src/common/helpers/format.helper.ts:1]: Fisierul ast ar trebui sters in the end right?

File: src/pages/programs/components/ProgramItem.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import { formatDate, formatDateYear } from '../../../common/helpers/Format.helper';
+import { IProgram } from './ProgramsList';
+
+const ProgramItem = ({ program }: { program: IProgram }) => {
+  const { t } = useTranslation('practice-programs');
+  return (
+    <Card>
+      <div className="flex sm:flex-row flex-col w-full lg:min-h-32 min-h-[6rem] items-center gap-x-8 gap-y-5 h-full">
+        <div className="aspect-square lg:w-32 sm:w-24 w-full lg:min-w-[8rem] sm:min-w-[6rem] bg-gray-100 sm:max-h-full max-h-[8rem]">
+          {program.image && <img className="w-full h-full bg-cover" src={program.image}></img>}
```
Comment (dragos1195) [inline src/pages/programs/components/ProgramItem.tsx:14]: Add "alt" for accessibility.

File: src/common/constants/nomenclature.constants.ts [hunk 3]
```diff
@@ -14,3 +14,62 @@ export const LANDING_COUNTER_ITEMS = [
   { value: '3500+', type: COUNTER_TYPE.PROGRAMS },
   { value: '500+', type: COUNTER_TYPE.NGOS },
 ];
+
+export const PRACTICE_PROGRAMS = {
```
Comment (birloiflorian) [inline src/common/constants/nomenclature.constants.ts:18]: Structura de practice program e diferita please check api

File: src/pages/programs/components/ProgramsList.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import ProgramItem from './ProgramItem';
+
+interface ProgramsListProps {
+  programs: IProgram[];
+  total: number;
+}
+
+export interface IProgram {
```
[the remark this answers is not in the document]
Comment (birloiflorian) [reply src/pages/programs/components/ProgramsList.tsx:10]: +1

**Review by birloiflorian — CHANGES_REQUESTED** [review]

File: src/common/helpers/format.helper.ts (hunk 1 above)
Comment (AndreeaMariaP) [reply src/common/helpers/format.helper.ts]: l-am sters

File: src/pages/programs/components/ProgramItem.tsx (hunk 2 above)
Comment (AndreeaMariaP) [reply src/pages/programs/components/ProgramItem.tsx]: am adaugat unde lipsea

File: src/pages/programs/components/ProgramsList.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (AndreeaMariaP) [reply src/pages/programs/components/ProgramsList.tsx]: am folosit interfata PracticeProgram

File: src/common/constants/nomenclature.constants.ts (hunk 3 above)
Comment (AndreeaMariaP) [reply src/common/constants/nomenclature.constants.ts:18]: am modificat

File: src/assets/locales/ro/translation.json [hunk 5]
```diff
@@ -50,6 +50,13 @@
       "title": "Domenii"
     }
   },
+  "practice-programs": {
```
Comment (birloiflorian) [inline src/assets/locales/ro/translation.json:53]: use underscore

File: src/pages/programs/components/ProgramItem.tsx [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+          <p className="font-titilliumBold sm:text-xl lg:text-3xl text-lg">{program.title}</p>
+          <div className="flex sm:flex-row flex-wrap gap-x-8 gap-y-2 sm:mt-auto">
+            <div className="flex flex-row items-center gap-x-1">
+              <LocationMarkerIcon className="w-4"></LocationMarkerIcon>
+              <p className="sm:text-sm lg:text-base text-xs">{program.location.name}</p>
+            </div>
+            <div className="flex flex-row items-center gap-x-1">
+              <CalendarIcon className="w-4"></CalendarIcon>
+              <p className="sm:text-sm lg:text-base text-xs">
+                {formatDate(program.startDate)} - {formatDate(program.endDate)}
```
Comment (birloiflorian) [inline src/pages/programs/components/ProgramItem.tsx:39]: end date poate sa vina null

**Review by birloiflorian — APPROVED** [review]

File: src/assets/locales/ro/translation.json (hunk 5 above)
Comment (AndreeaMariaP) [reply src/assets/locales/ro/translation.json]: fixed

File: src/pages/programs/components/ProgramItem.tsx (hunk 6 above)
Comment (AndreeaMariaP) [reply src/pages/programs/components/ProgramItem.tsx:39]: checked

**Pull request merged**